hangman/lib/database.py

```diff
@@ -0,0 +1,19 @@
+import sqlite3
+
+
+contents = sqlite3.connect("hangman/data/store_guesses.sqlite")
+contents.execute("CREATE TABLE IF NOT EXISTS guesses(letter CHAR)")
+
+def guesses_table(guess):
+    """ adds the guesses into the table created"""
+    contents.execute("INSERT INTO guesses VALUES('{}')".format(guess))
+    cursor = contents.cursor()
+    cursor.execute("SELECT * FROM guesses")
+    print(cursor.fetchall())
+    contents.commit()
```

Comment:
The question didn't ask for this function. You're to just  create and drop the table for now

Reply:
So, I shouldn't insert values to the table?

Reply:
Since you've already done it, just leave it.